core/payment/tests/test_allocation.rs

```diff
@@ -161,6 +161,145 @@ async fn test_release_allocation(ctx: &mut DroppableTestContext) -> anyhow::Resu
     Ok(())
 }
 
+#[cfg_attr(not(feature = "system-test"), ignore)]
+#[test_context(DroppableTestContext)]
+#[serial_test::serial]
+async fn test_auto_release_allocation(ctx: &mut DroppableTestContext) -> anyhow::Result<()> {
+    enable_logs(false);
+
+    let dir = temp_dir!("test_release_allocation")?;
```

Comment:
```suggestion
    let dir = temp_dir!("test_auto_release_allocation")?;
```
Otherwise it will create directory under wrong name and it will be harder to find.